docs/components/board/_index.md

```diff
@@ -0,0 +1,1534 @@
+---
+title: "Board Component"
+linkTitle: "Board"
+weight: 20
+type: "docs"
+no_list: true
+description: "The signal wire hub of a robot, with GPIO pins for transmitting signals between the robot's computer and its other components."
+tags: ["board", "components"]
+icon: "/components/img/components/board.svg"
+# SMEs: Gautham, Rand
+---
+
+A *board* is the signal wire hub of a robot that provides access to general purpose input/output [(GPIO)](https://www.howtogeek.com/787928/what-is-gpio/) pins: a collection of pins on the motherboard of a computer that can receive electrical signals.
```

Comment:
General comment: I am not opposed to adding the diagram and other information about viam-server and computers back in here, but I think this helps greatly in addressing shawns SE feedback on this page (https://viam.atlassian.net/browse/DOCS-499) that it gets too complex too quickly. 

Also it doesn't really make sense that a computer + GPIO peripheral would be a *board* itself, which the previous intro implied. 
Also the new `pca9685` model's pins just support PWM signaling and I assume that more peripheral types will be added into this component in the future? 

I would be open to make part of this pr moving the diagram elsewhere but am not sure where

Reply:
Why wouldn't that be a board?

Reply:
Because the computer is not a board and is not part of board configuration